### ISSUE TITLE
DigitalOcean machine pool blank for rke2/k3s clusters

### ISSUE THREAD

**Issue by dnoland1** [rancher/dashboard#8747]
**Setup**
- Rancher version: 2.7.2
- Browser type & version: Firefox 112.0 (Mac)

**Describe the bug**
Machine pool is blank:
![image](https://user-images.githubusercontent.com/468462/234714449-a1f34233-d406-4fac-861c-7a19718e4bcb.png)

**To Reproduce**
Go to Cluster Manager
Click Create button
Toggle to rke2/k3s clusters
Select DigitalOcean

**Result**
The machine pool is blank (see screen shot above)

**Expected Result**
Fields for configuring the machine pool

**Screenshots**
See above

**Additional context**
Saw this error in browser console which might be related:
```
TypeError: e is undefined
    configComponent MachinePool.vue:118
    VueJS 3
    ge MachinePool.vue:1
    VueJS 58
    fetch SelectCredential.vue:58
    n NuxtJS
    v runtime.js:63
    _invoke runtime.js:294
    I runtime.js:119
    Babel 4
    NuxtJS 3
    VueJS 19
console.js:31:30
```

**Comment by richard-cox** [comment]
This is as odd one, DO is often used throughout development. Do you have any extensions installed? Could you try again with new credentials using a verified API key?

**Comment by dnoland1** [comment]
Are you not able to reproduce? Two people on my support team reported the issue to me and I'm able to reproduce it (we all have different keys). You can also give it a go at support.rancher.cloud. I see OS Management in the menu, so I think that means the Elemental extension is installed.

**Comment by richard-cox** [comment]
What version of the elemental ui extension is installed, and would you be able to temporarily disable it and try again?

**Comment by dnoland1** [comment]
Sorry, I saw it in the menu, but when I click on it (I'm an admin), does not appear it's installed:
![image](https://user-images.githubusercontent.com/468462/235177936-66bfbf0d-ac7e-459f-b6a4-ea0f5ff67034.png)

**Comment by richard-cox** [comment]
Can you go to the Burger menu --> `Extensions` page and let me know which extensions are installed? They can be separate from the product underneath. For the elemental extension it will guide you through the install process for elemental actual (as well as show elemental features when elemental actual is installed)

**Comment by dnoland1** [comment]
No Extensions on my Burger:
![image](https://user-images.githubusercontent.com/468462/235180741-eb226bf8-b997-463f-887b-695dee598eae.png)

**Comment by dnoland1** [comment]
I am an admin, Burger is broken and get the above console error when I click on the Burger. I'm logged in with username=admin with full Administrator rights. I can also check things in the backend if needed.

**Comment by richard-cox** [comment]
it looks like for a `provisioning.cattle.io.clusters` we can't find the associated `management.cattle.io.cluster` . are there any any failed network requests, or clusters stuck in an odd deleted state?

**Comment by dnoland1** [comment]
Yes, found an old cluster that had been deleted that had a resource in cluster.provisioning.cattle.io, but not cluster.management.cattle.io. I've removed it, but still facing the same issues with the burger menu as well as DigitalOcean machine pool.

**Comment by dnoland1** [comment]
Sorry, deleted it, but was stuck in "deleting" state. It's now gone and I can get to the Burger and see extensions:
<img width="1228" alt="image" src="https://user-images.githubusercontent.com/468462/235203808-02a3e6de-18d7-458d-a713-858e1428f286.png">

I'll update Elemental to 1.1.0 and see if that helps.

**Comment by dnoland1** [comment]
Looks like that fixed the issue:
<img width="1026" alt="image" src="https://user-images.githubusercontent.com/468462/235204288-595c4803-993f-4da3-9568-4aa8adb82685.png">
Anything that can be done so that Extensions don't break core application functionality?

**Comment by nwmac** [comment]
I believe the issues causing this issue have been resolved, so marking as resolved in 2.11. Please re-open if this is still an issue for you.
